### PR TITLE
Updates for this workshop

### PR DESCRIPTION
**Pull request by boshek**

### CODE REVIEW

File: intro.qmd [hunk 1]
```diff
@@ -39,24 +38,6 @@ Posit is committed to providing a professional, friendly and safe environment fo
 | 15:00 - 15:30 | *Coffee break*                                                |
 | 15:30 - 17:00 | Continuous Integration; Package Website; Package Distribution |
 
-### Sticky notes
```
[the remark this answers is not in the document]
Comment (ateucher) [reply intro.qmd:42]: I jest but we should have away to signal ok vs help virtually

File: style-dark.scss [hunk 2]
```diff
@@ -1,10 +1,10 @@
 /*-- scss:defaults --*/
 @import 'https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible&display=swap';
-@import 'https://fonts.googleapis.com/css2?family=Fira+Code:wght@300;400;500;600;700&display=swap';
+@import 'https://fonts.googleapis.com/css2?family=DM+Mono:wght@300;400;500;600;700&display=swap';
```
Comment (ateucher) [inline style-dark.scss:3]: Why the font switch?

**Review by ateucher — APPROVED** [review]
Thanks! Looks good - maybe a couple of things to discuss before merging?

File: intro.qmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (boshek) [reply intro.qmd:42]: Made an issue. Agree but we can defer that part for now.

File: style-dark.scss (hunk 2 above)
Comment (boshek) [reply style-dark.scss:3]: No reason but upon further reflection I think Fira looks better here.

**Pull request merged**